opm/grid/cpgrid/processEclipseFormat.cpp

```diff
@@ -239,6 +239,36 @@ namespace cpgrid
             // Make the grid.
             processEclipseFormat(g, nnc_cells, z_tolerance, false, turn_normals);
         }
+
+        {
+            auto format_ijk = [this](int cell) -> std::string {
+                std::array<int,3> ijk;
+                this->getIJK(cell, ijk);
+                return std::string{"("} + std::to_string(ijk[0]) + std::string{","} + std::to_string(ijk[1]) + std::string{","} + std::to_string(ijk[2]) + std::string{")"};
```

Comment:
Just a tangential note:  `CpGridData::getIJK()` returns zero-based IJK tuples.  Do we want to report them to the user as one-based indices (i.e., `std::to_string(ijk[0] + 1)` and so on)